roles/system/templates/deployments/system-app-deployment.yaml

```diff
@@ -0,0 +1,468 @@
+          imagePullPolicy: "{{ image.pull_policy | default(image_pull_policy) }}"
+          args:
+            - env
+            - PORT=3000
+            - container-entrypoint
+            - bundle
+            - exec
+            - unicorn
+            - -c
+            - config/unicorn.rb
+        ## container > system-app > ports
+          ports:
+            - containerPort: 3000
+              name: ui-api
+              protocol: TCP
+            - containerPort: 9394
+              name: metrics
+              protocol: TCP
+        ## container > system-app > resources
+          resources:
+            requests:
+              memory: "{{ app.resources.requests.memory | default(app_resources_requests_memory) }}"
+              cpu: "{{ app.resources.requests.cpu | default(app_resources_requests_cpu) }}"
+            limits:
+              memory: "{{ app.resources.limits.memory | default(app_resources_limits_memory) }}"
+              cpu: "{{ app.resources.limits.cpu | default(app_resources_limits_cpu) }}"
+        ## container > system-app > probes
+          livenessProbe:
+            tcpSocket:
+              port: ui-api
+            initialDelaySeconds: {{ app.liveness_probe.initial_delay_seconds | default(app_liveness_probe_initial_delay_seconds) }}
+            timeoutSeconds: {{ app.liveness_probe.timeout_seconds | default(app_liveness_probe_timeout_seconds) }}
+            periodSeconds: {{ app.liveness_probe.period_seconds | default (app_liveness_probe_period_seconds) }}
+            successThreshold: {{ app.liveness_probe.success_threshold | default(app_liveness_probe_success_threshold) }}
+            failureThreshold: {{ app.liveness_probe.failure_threshold | default(app_liveness_probe_failure_threshold) }}
+          readinessProbe:
+            httpGet:
+              httpHeaders:
+                - name: X-Forwarded-Proto
+                  value: https
+              path: /check.txt
+              port: ui-api
+              scheme: HTTP
+            initialDelaySeconds: {{ app.readiness_probe.initial_delay_seconds | default(app_readiness_probe_initial_delay_seconds) }}
+            timeoutSeconds: {{ app.readiness_probe.timeout_seconds | default(app_readiness_probe_timeout_seconds) }}
+            periodSeconds: {{ app.readiness_probe.period_seconds | default (app_readiness_probe_period_seconds) }}
+            successThreshold: {{ app.readiness_probe.success_threshold | default(app_readiness_probe_success_threshold) }}
+            failureThreshold: {{ app.readiness_probe.failure_threshold | default(app_readiness_probe_failure_threshold) }}
+        ## container > system-app > volumeMounts
+          volumeMounts:
+            - mountPath: /opt/system-extra-configs
+              name: system-config
+              readOnly: true
+        ## container > system-app > env
+          env:
+            - name: THINKING_SPHINX_ADDRESS
+              value: {{ sphinx.env.thinking.endpoint | default("system-sphinx") }}
```

Comment:
```suggestion
              value: {{ sphinx.env.thinking.endpoint | default(sphinx_env_thinking_endpoint) }}
```

Reply:
This was intentional, as by default will use the `system-sphinx` service (the service object name) created by the SaaS operator.